fedmsg_meta_fedora_infrastructure/pkgdb.py

```diff
@@ -106,87 +111,102 @@ def subtitle(self, msg, **config):
             agent = get_agent(msg)
 
             try:
+                namespace = msg['msg']['package_listing']['package'].get('namespace', 'rpms')
```

Comment:
Ah, it will if `'package_listing'` is not in `msg['msg']`.  Look at the `except:` block and see that this is the case being handled by the original code.